projects/plugins/jetpack/extensions/editor.js

```diff
@@ -78,10 +78,16 @@ function setBetaBlockTitle( settings, name ) {
 		return settings;
 	}
 
+	const { title, keywords } = settings;
+	const titleSuffix = '(beta)';
+	const betaKeyword = 'beta';
+
 	return {
 		...settings,
-		title: `${ settings.title } (beta)`,
-		kewords: [ ...settings.keywords, 'beta' ],
+		title: title.toLowerCase().endsWith( titleSuffix )
```

Comment:
Some blocks have a `name` suffixed by `(beta)` in their `block.json` file. This makes sure the 'beta' mention doesn't appear more than once.

Reply:
👍 Maybe we can remove the (beta) suffix from the block.json file entries now, and then land this so we don't need the extra logic?

Reply:
The beta mention in the `block.json` name and description is necessary if we want to display it in [the Jetpack plugin wp.org page](https://wordpress.org/plugins/jetpack/). wp.org only parses `block.json` to generate the blocks listing.

Reply:
Ah right, I forgot about that!